sample/src/main.rs

```diff
@@ -133,6 +129,11 @@ async fn main() {
         time.as_nanos() as u32,
     ));
 
+    println!(
+        "Node public key: {}",
+        km.get_node_id(lightning::sign::Recipient::Node).unwrap()
+    );
```

Comment:
Oh yeah, this is that sneaky bug fix you mentioned a while ago.

Reply:
Yeah, wasted way to much time on that...